kalliope/tts/watson/watson.py

```diff
@@ -0,0 +1,62 @@
+from kalliope.core.Utils.FileManager import FileManager
+from requests.auth import HTTPBasicAuth
+from kalliope.core.TTS.TTSModule import TTSModule
+import logging
+import requests
+
+
+logging.basicConfig()
+logger = logging.getLogger("kalliope")
+
+TTS_URL = "https://stream.watsonplatform.net/text-to-speech/api/v1"
+TTS_CONTENT_TYPE = "audio/wav"
+
+
+class Watson(TTSModule):
```

Comment:
As parameters as required, can you check parameters and raise an exception if one is missing?